.dependabot/config.yml

```diff
@@ -10,3 +10,14 @@ update_configs:
     # pull requests
     default_reviewers:
       - "ministryofjustice/crime-billing-online"
+
+  # Keep package.json (& lockfiles) up to date as soon as
+  # new versions are published to the npm registry
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "daily"
```

Comment:
might be worth doing this weekly, same for gem dependencies as circleci gets overloaded. see #circleci-users slack channel for more on that.

I am not sure how much control we have via this config as their are some organisation level dependabot config coming into play

see https://app.dependabot.com/accounts/ministryofjustice